.gitignore

```diff
@@ -1,20 +1,25 @@
 .DS_Store
```

Comment:
What happened here?

Reply:
I added entries to ignore my test package, then removed it but left it in this state... I can revert if you want it to.

Reply:
please if you can , reviewing is easier when unrelated noise is removed